centos/7/Dockerfile

```diff
@@ -19,6 +19,7 @@ RUN yum -y install \
     yum-utils
 RUN yum -y install epel-release centos-release-scl centos-release-scl-rh
 RUN curl -s https://packagecloud.io/install/repositories/packpack/backports/script.rpm.sh | bash
+RUN find /etc/yum.repos.d/ -type f -exec sed -i 's/packagecloud.io\/packpack/packpack.hb.bizmrg.com/g' {} +
```

Comment:
This code still leans to some extent on packagecloud.io. I would rather place a ready `*.repo` file into the directory.

Reply:
It seems, now it is fixed.